core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -227,6 +232,36 @@
     top: 14px;
   }
 
+  .conversation-skin-tutor-card-tooltip {
+    padding-top: 8px;
+    display: inline-block;
+  }
+
+  .conversation-skin-tutor-card-tooltip-button {
+    position: relative;
+    border-radius: 50%;
+    background-color: #ff0000;
+    padding-top: 4px;
+    text-align: center;
+    font-size: 12px;
+    color: #ffffff;
+    font-weight: 900;
+  }
+
+  .tooltip-text {
```

Comment:
This class is too general; prefix it by something that scopes it to the conversation skin (like you've done in the rule below).

Reply:
Done!